flare/examples/img_q_example.py

```diff
@@ -0,0 +1,82 @@
+import torch.nn as nn
```

Comment:
Please make sure that each example can produce a reasonable training curve.

Reply:
The produced curves seem comparable with the left part of AC curve.